src/components/organisms/Footer.tsx

```diff
@@ -7,11 +7,11 @@ import './Footer.scss'
 export class Footer extends React.Component<undefined, undefined> {
   render() {
     return (
-      <footer>
-        <div className="container">
-          <div className="logo-and-social">
+      <footer className="footer">
```

Comment:
Why do we want to add a class `footer` to an element `footer`?

Reply:
Because I left the footer in for the feature toggle, it wasn't allowing me to style both of them differently. We could change it back if we aren't going to use the feature toggle.

Reply:
Got it. 

Some background on this:

We never made a solid decision on whether or not to use BEM - Walter decided to use it in frost-web, while I was simply selecting elements by their names, avoiding using classes as much as possible. 

As far as I remember, Walter doesn't like it either, but I hadn't provided any specifications on how to write styles and that was what he believed was most standard.

My reasoning is that CSS classes make sense in the traditional CSS mindset, where all CSS is global, but when component encapsulation and semantic HTML elements are introduced, selecting by HTML element names makes more sense. 

For example: any given element can have a header or a footer, but no single element should have more than one. And, given all the semantic HTML 5 elements, we should almost never need to use divs directly.

Anyway, the conclusion from this is: for now, whichever styling pattern you prefer is fine, but let's avoid migrating existing files from one to the other until we decide to stick with one.

@kennylavender @wzalazar feedback?

Reply:
I agree with trying to use semantic elements whenever possible. I think whether or not we can possibly use mostly semantic elements for styling will be design dependent. All responsive designs I've had to build in the past always required lots of extra div markup, so I always used class names even with scoped CSS. Can't really speak to the pros/cons of either direction at the moment.